crates/ruma-macros/src/api/response/outgoing.rs

```diff
@@ -67,7 +68,10 @@ impl Response {
                 fn try_into_http_response<T: ::std::default::Default + #bytes::BufMut>(
                     self,
                 ) -> ::std::result::Result<#http::Response<T>, #ruma_common::api::error::IntoHttpError> {
+                    use #http::StatusCode;
+
                     let mut resp_builder = #http::Response::builder()
+                        .status(StatusCode::#status_ident)
```

Comment:
We tend not to use imports in the generated code.

```suggestion
                    let mut resp_builder = #http::Response::builder()
                        .status(#http::StatusCode::#status_ident)
```